rpc-server/src/modules/queries/contract_runner/code_storage.rs

```diff
@@ -46,19 +46,21 @@ impl CodeStorage {
             readnode_primitives::StateKey,
             Option<readnode_primitives::StateValue>,
         >,
+        storage_usage: u64,
+        prefetch_state_size_limit: u64,
     ) -> Self {
-        let prefetch_state_data = if account_id.to_string().ends_with("poolv1.near") {
-            utils::get_state_from_db(
+        let mut prefetch_state_data = HashMap::new();
+
+        if storage_usage < prefetch_state_size_limit {
```

Comment:
```rust
let prefetch_state_data = if storage_usage < prefetch_state_size_limit {
     utils::get_state_from_db(
        &db_manager,
        &account_id,
        block_height,
        &[],
        "query_call_function",
    )
    .await
} else {
    HashMap::new()
}

Reply:
Good catch, thanks! I forgot to change this after removing second request for account info